### DISCUSSION TITLE
Clear feed activity from people you unfollowed

### DISCUSSION THREAD

**Discussion by RobertoFDuran** [community/community#30839] [Other Feature Feedback, Questions, & Ideas]
Has there been any update on how to clear the past activity of someone you unfollowed? It still shows up on my homepage and it makes it hard to keep my stuff organized.

**Comment by scruel** [comment]
+1, is this feature really too hard to implement?

**Comment by jfullstackdev** [comment]
I'm seeing a new update from the team 

![image](https://user-images.githubusercontent.com/47092464/194704787-6c03967b-cda5-41c7-b165-ed70e5ca4f9e.png)

maybe it will help, although again, it's still beta
